examples/tree_2d_dgsem/elixir_navierstokes_shearlayer_amr.jl

```diff
@@ -5,12 +5,11 @@ using Trixi
 ###############################################################################
 # semidiscretization of the compressible Navier-Stokes equations
 
-# TODO: parabolic; unify names of these accessor functions
 prandtl_number() = 0.72
-mu() = 1.0 / 3.0 * 10^(-5) # equivalent to Re = 30,000
+mu(u, equations) = 1.0 / 3.0 * 10^(-5) # equivalent to Re = 30,000
```

Comment:
```suggestion
mu(u, equations) = 1.0 / 3.0 * 10^(-4) # equivalent to Re = 30,000
```
Unrelated to the current PR, but I think this is off and is actually setting the Reynolds number to be 300,000. If we take Re = 30,000 then we would have mu = 1/30000 ≈ 3.3333e-5 but with this we have mu = 1/3 * 10^(-5) ≈ 3.3333 * 10^(-1) * 10^(-5) = 3.3333 * 10^(-6).

Reply:
Ah yeah right. I will update the test values accordingly.